File: 1032/main.cs
using System;

class Program
{
    static void Main()
    {
        // Solicitar ao usuário a quantidade de elementos a serem adicionados ao 
        //array
        Console.Write("How many elements to add to the array: ");
        int n = int.Parse(Console.ReadLine());

        // Inicializar arrays para armazenar os elementos e as somas prefixas
        int[] arrayForInput = new int[n];
        int[] arrayForManipu = new int[n + 1];
        
        // Receber os elementos do usuário, calcular as somas prefixas e armazenar 
        //os elementos fornecidos
        int s = 0;
        arrayForManipu[0] = 0;
        for (int i = 0; i < n; i++)
        {
            Console.Write($"[{i}] = ");
            arrayForInput[i] = int.Parse(Console.ReadLine());
            s += arrayForInput[i];
            arrayForManipu[i + 1] = s;
        }

        // Iterar sobre as somas prefixas para encontrar um subconjunto cuja 
        //soma é um múltiplo de n
        for (int i = 0; i < n + 1; i++)
        {
            for (int j = 0; j < i; j++)
            {
                if ((arrayForManipu[i] - arrayForManipu[j]) % n == 0)
                {
                    // Exibir o tamanho do subconjunto e seus elementos
                    int subsetSize = i - j;
                    Console.WriteLine($"Size of subset: {subsetSize}");
                    Console.Write("Subset = [");
                    for (int k = j; k < i; k++)
                    {
                        Console.Write($" {arrayForInput[k]} ");
                    }
                    Console.WriteLine($"] => {n}");
                    return;
                }
            }
        }

        // Se nenhum subconjunto for encontrado
        Console.WriteLine("0");
    }
}

File: 1032/main.py
# Solicitar ao usuário a quantidade de elementos a serem adicionados ao array
n = int(input("How many elements to add to the array: "))

# Inicializar listas para armazenar os elementos e as somas prefixas
array_for_input = []
array_for_manipu = []

# Receber os elementos do usuário, calcular as somas prefixas e armazenar os 
#elementos fornecidos
s = 0
for i in range(n):
    element = int(input(f"[{i}] = "))
    array_for_input.append(element)
    s += element
    array_for_manipu.append(s)

# Iterar sobre as somas prefixas para encontrar um subconjunto cuja soma é um 
#múltiplo de n
for i in range(n + 1):
    for j in range(i):
        if (array_for_manipu[i] - array_for_manipu[j]) % n == 0:
            # Exibir o tamanho do subconjunto e seus elementos
            subset_size = i - j
            print(f"Size of subset: {subset_size}")
            print("Subset =", array_for_input[j:i])
            print("=>", n)
            exit(0)

# Se nenhum subconjunto for encontrado
print("0")
